soh/soh/GameMenuBar.cpp

```diff
@@ -815,8 +815,10 @@ namespace GameMenuBar {
                     UIWidgets::Tooltip("Instantly return the boomerang to Link by pressing its item button while it's in the air");
                     UIWidgets::PaddedEnhancementCheckbox("Prevent Dropped Ocarina Inputs", "gDpadNoDropOcarinaInput", true, false);
                     UIWidgets::Tooltip("Prevent dropping inputs when playing the ocarina quickly");
-                    UIWidgets::PaddedEnhancementCheckbox("MM Bunny Hood", "gMMBunnyHood", true, false);
-                    UIWidgets::Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask");
+                    UIWidgets::PaddedText("MM Bunny Hood", true, false);
+                    const char* bunnyHoodOptions[3] = { "Disabled", "Fast + Jump Velocity", "Fast"};
+                    UIWidgets::EnhancementCombobox("gMMBunnyHood", bunnyHoodOptions, 3, 0);
+                    UIWidgets::Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask. The jump velocity boost is not accounted for in randomizer logic.");
```

Comment:
My suggestion would be to rename `MM Bunny Hood` to `Fast Bunny Hood`, as without the jump distance, this isn't actually MM bunny hood anymore, and would convey the intention a bit clearer.

And this might be subjective, but I'd prefer `"Disabled", "Ground + Jump speed", "Ground speed"` for the option names.

Reply:
I agree here expect for “jump speed”, as it is more about the velocity, but I’m not sure what would be clearer for the general pop

Reply:
"Ground Speed & Jump Distance"? Not sure if that's becoming too long or not.

Reply:
What about:
* `Disabled`
* `Run`
* `Run + Jump`

Reply:
That works for me.